### DISCUSSION TITLE
Feature suggestion: multiline custom events props

### DISCUSSION THREAD

**Discussion by janosh** [plausible/analytics#1016] [Anything Else]
Would be cool if the web UI allowed for line breaks in custom events props. In our case, we're recording if form submissions fail and in that case sending along the error's stack trace.

![Screen Shot 2021-05-10 at 16 25 02](https://user-images.githubusercontent.com/30958850/117675456-de7a9980-b1ac-11eb-8b55-f0535eac0f71.png)

**Comment by blyme** [comment]
I've created a suggestion on their feature request board, linking to this discussion
https://feedback.plausible.io/268
